pkg/scheduler/metrics/metrics.go

```diff
@@ -72,17 +70,6 @@ var (
 		},
 		[]string{OperationLabel},
 	)
-	DeprecatedSchedulingLatency = prometheus.NewSummaryVec(
```

Comment:
We cannot remove the scheduler metrics. These are needed for backward compatibility. We must keep them for at least a year after deprecation (1.14).

Reply:
@bsalamat  you mean all scheduler metrics must be kept ?
cc @brancz

Reply:
Metrics are not covered by stability agreements. Unless there is a pressing issue like the kubernetes scalability tests themselves, I don't think we should be making further exceptions. The intention of changing these metrics was long communicated and so was the removal.